tokens/source/components/status.json

```diff
@@ -0,0 +1,341 @@
+{
+  "status": {
+    "color": {
+      "1na": {
+        "comment": "Analytics unknown 1xx request status color (blue.10).",
```

Comment:
If you all refer to it as `1xx` why is it named `1na`?

Reply:
I'm just doing my best to appease both camps by using the `1xx` wildcard terminology in the comments, to appease those who didn't vote for `-na` ;)

Reply:
Tbh I think it makes it more confusing…